docker/autoware-universe/Dockerfile

```diff
@@ -53,14 +53,70 @@ RUN chmod 644 /etc/glvnd/egl_vendor.d/10_nvidia.json
 RUN echo "source /opt/ros/${ROS_DISTRO}/setup.bash" > /etc/bash.bashrc
 CMD ["/bin/bash"]
 
-FROM devel as prebuilt
+FROM devel as builder
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 ## Build and change permission for runtime data conversion
 RUN source /opt/ros/"$ROS_DISTRO"/setup.bash \
   && colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release \
   && find /autoware/install -type d -exec chmod 777 {} \;
 
+ARG CUDA_RUNTIME_BASE_IMAGE
+# hadolint ignore=DL3006
+FROM $CUDA_RUNTIME_BASE_IMAGE as prebuilt
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG ROS_DISTRO
+ARG SETUP_ARGS
+
+## Install apt packages
+# hadolint ignore=DL3008
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+  git \
+  ssh \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+## Copy files
+COPY autoware.repos setup-dev-env.sh ansible-galaxy-requirements.yaml amd64.env arm64.env /autoware/
+COPY ansible/ /autoware/ansible/
+WORKDIR /autoware
+RUN ls /autoware
```

Comment:
Is this for debugging?
```suggestion
```

Reply:
It is copied from devel build process. Line 21

Reply:
Oh, sorry. I forgot about that. :sweat_smile: 
I will recheck if it's necessary by seeing the CI log.

Reply:
It might be useful for debugging because we can check what files are included.
Also, as far as I know, there is no alternative way to print debug messages and it doesn't increase the generated image size.
So I'll keep it.